sunpy/instr/rhessi.py

```diff
@@ -219,7 +219,55 @@ def get_obssum_filename(time_range):
 
     filenames = dbase_dat.get('filename')[index_number_start:index_number_end]
     return [posixpath.join(get_base_url(), 'metadata', 'catalog', filename + 's')
-            for filename in filenames]
+        for filename in filenames]
+
+
+def get_obssum_filename_multiple_months(time_range):
+    """
+    Download the RHESSI observing summary data from one of the RHESSI
+    servers, parses it, and returns the name of the obssumm files relevant for
+    the time range.
+
+    Parameters
+    ----------
+    time_range : str, TimeRange
```

Comment:
@Cadair is this a standard that we always also accept a string version of a time range?